kubectl-cloudflow/config/config.go

```diff
@@ -335,23 +335,23 @@ func validateLabels(podConfig *configuration.Config, podName string) error {
 		}
 
 		for key, value := range labelsConfig.Root().GetObject().Items() {
-			label := strings.TrimSpace(key)
+			labelKey := strings.TrimSpace(key)
 			labelValue := strings.TrimSpace(value.String())
 
-			if labelHasPrefix(label) {
-				splitted := strings.Split(label, "/")
+
+			if labelHasPrefix(labelKey) {
+				splitted := strings.Split(labelKey, "/")
 				prefix := splitted[0]
-				label := splitted[1]
+				suffix := splitted[1]
 
-				if err := validateLabel(label, prefix); err != nil {
+				if err := validateLabelPrefix(labelKey, prefix); err != nil {
 					return err
 				}
-				return validateLabelValue(labelValue, label)
-			}
-			if err := validateLabel(label, ""); err != nil {
+				return validateLabelValue(labelValue, suffix)
+			} else if err := validateLabelValue(labelKey, ""); err != nil {
```

Comment:
I don;t get this part, why is it necessary to do validateLabelValue with a "" label, validating the labelKey? that is very confusing.

Reply:
Because there are two possible validation in the labelKey, keys that have prefix, and keys that don't have it. Makes sense?